HerculesStudio-manifest.yaml

```diff
@@ -0,0 +1,21 @@
+---
+app-id: com.jacobdekel.HerculesStudio
+runtime: org.kde.Platform
+runtime-version: '6.3'
+sdk: org.kde.Sdk
+command: HerculesStudio
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
+- "--socket=wayland"
+- "--filesystem=host"
+- "--device=dri"
+modules:
+- name: Hercules-Studio
+  buildsystem: cmake
+  subdir: HerculesStudio
+  sources:
+  - type: git
+    url: file:///home/yakov/hercstudio-git-code
```

Comment:
still won't be able to find the source.

Reply:
Auch... committed the wrong file... SORRY!